R/formatter.r

```diff
@@ -1,6 +1,12 @@
-#' Number formatter: a generic formatter for numbers
+#' Number formatters
 #'
-#' @return `number_format` returns a function with single parameter
+#' `number_format()` and `number()` are generic formatters for numbers.
```

Comment:
Maybe add something like: ``"All formatters allow you to re-`scale` (multiplicatively) and round to specified `accuracy`"``

Reply:
done